### DISCUSSION TITLE
What's the difference between lazy pack 1080p 1X and 4K 2X?

### DISCUSSION THREAD

**Discussion by shuaihuaiyi** [HolographicWings/TOTK-Mods-collection#118] [Q&A] [answered]
If the sole distinction between the two packs is setting the scale at 1x or 2x in Yuzu, what is the need for separate packs? Moreover, the readme states that "1080p (1920x1080) when scaled above 1x resolution will break Ambient Occlusion." Does this mean that using 4K 2X will now cause Ambient Occlusion to break?

**Comment by hoverbike1** [comment, marked as answer]
the difference between 1080p x1 and 4k 2x/8k 4x is that
one is made for 1x scaling, and the other is made for scaling above 1x
certain mods cause graphical artifacts when not used correctly, for instance, sky island fix at 1x. this results in artifacts in the sky.

therefore, there are scaling, and non-scaling versions, to avoid this issue.
